services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/QrCodePosterTemplateStructureProvider.java

```diff
@@ -70,7 +68,7 @@ private QRCodePosterTemplateAndroid buildAndroidProtoStructure(
         .setTemplate(qrTemplateLoader.loadAndroidTemplateAsBytes())
         .setQrCodeSideLength(templateConfig.getQrCodeSideLength())
         .setDescriptionTextBox(QRCodeTextBoxAndroid.newBuilder()
-            .setOffsetX(textBoxConfig.getOffsetX().floatValue()).setOffsetY(textBoxConfig.getOffsetY().floatValue())
+            .setOffsetX(textBoxConfig.getOffsetX().intValue()).setOffsetY(textBoxConfig.getOffsetY().intValue())
```

Comment:
Didn't Android want to have the float? (see [here](https://github.com/corona-warn-app/cwa-server/pull/1314/files#diff-45fb40b80c6e2f60b43366005db12e9bf4ff8c288645ad642988316e990e7ab5R68))

Reply:
I got a compile error